deepchem/models/optimizers.py

```diff
@@ -808,3 +808,54 @@ def _create_pytorch_optimizer(self):
         else:
             self.kwargs['lr'] = self.learning_rate
         return KFACOptimizer([self.kwargs])
+
+
+class Lamb(Optimizer):
+    """The Lamb optimization algorithm."""
+
+    def __init__(self,
+                 learning_rate: Union[float, LearningRateSchedule] = 0.001,
+                 beta1: float = 0.9,
+                 beta2: float = 0.999,
+                 epsilon: float = 1e-08,
+                 weight_decay: float = 0):
+        """Construct an Adam optimizer.
+
+        Parameters
+        ----------
+        learning_rate: float or LearningRateSchedule
+            the learning rate to use for optimization
+        beta1: float
+            a parameter of the Lamb algorithm
+        beta2: float
+            a parameter of the Lamb algorithm
+        epsilon: float
+            a parameter of the Lamb algorithm
+        weight_decay: float
+            L2 penalty - a parameter of the Lamb algorithm
+
+        References
+        ----------
+        Yang You and Jing Li and Sashank Reddi, et, al., "Large Batch Optimization for Deep Learning: Training BERT in 76 minutes", 2020,
+        https://arxiv.org/abs/1904.00962
+        """
+        super(Lamb, self).__init__(learning_rate)
+        self.beta1 = beta1
+        self.beta2 = beta2
+        self.epsilon = epsilon
+        self.weight_decay = weight_decay
+
+    def _create_pytorch_optimizer(self, params):
+        try:
+            import torch_optimizer
+        except ModuleNotFoundError:
+            raise Exception('This optimizer requires torch-optimizer module.')
+        if isinstance(self.learning_rate, LearningRateSchedule):
+            lr = self.learning_rate.initial_rate
+        else:
+            lr = self.learning_rate
+        return torch_optimizer.Lamb(params,
```

Comment:
Let's try to shift the implementation into DeepChem. torch_optimizer doesn't look to be actively maintained